app/sidekiq/inning_rollover_job.rb

```diff
@@ -0,0 +1,11 @@
+class InningRolloverJob
+  include Sidekiq::Job
+
+  def perform(inning_id)
+    Inning.current_to_false
+    inning = Inning.find(inning_id)
+    inning.update(current: true)
```

Comment:
Let's refactor this to use the `Inning#make_current_inning` method. I like the idea of the `current_to_false` method, it's just duplicating the logic in the `make_current_inning` method.

Reply:
Updated changes